apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagService.kt

```diff
@@ -287,6 +305,21 @@ class RealGrunnlagService(
     )
 }
 
+private fun HistorikkForeldreansvar.tilGrunnlagsopplysning(
+    fnr: Folkeregisteridentifikator
+): Grunnlagsopplysning<JsonNode> {
+    return Grunnlagsopplysning(
+        id = UUID.randomUUID(),
+        kilde = Grunnlagsopplysning.Pdl(Tidspunkt.now(), null, null),
+        opplysningType = Opplysningstype.HISTORISK_FORELDREANSVAR,
+        meta = object {}.toObjectNode(),
```

Comment:
Ikke nullable?

Reply:
Nei, men endret til samme framgangsmåte brukt ellers for å lage tom meta